redash/handlers/users.py

```diff
@@ -152,6 +152,28 @@ def post(self, user_id):
         }
 
 
+class UserRegenerateApiKeyResource(BaseResource):
+    def post(self, user_id):
+        user = models.User.get_by_id_and_org(user_id, self.current_org)
+        if user.is_disabled:
+            abort(404, message='Not found')
+        if not is_admin_or_owner(user_id):
+            abort(403, message="You cannot regenerate API Key."
```

Comment:
I think just `abort(403)` is enough. We will not show this UI to non admins once #3234 is resolved.